publication-request.json

```diff
@@ -2,12 +2,14 @@
     "package-id" : "ans.hl7v2.fr.trans-cda-r2",
     "version" : "2.1.1",
     "path" : "https://interop.esante.gouv.fr/ig/hl7v2/trans-cda-r2/2.1.1",
-    "mode" : "working",
-    "status" : "draft",
-    "sequence" : "trial-use",
-    "first": true,
-    "title" : "",
-    "desc" : "",
+    "mode" : "milestone",
+    "status" : "trial-use",
```

Comment:
```suggestion
    "status" : "trial-implementation",
```

Reply:
j'ai suivi la doc :) "status" : the status of this release, one of release|trial-use|update|preview|ballot|draft|normative+trial-use|normative|informative https://confluence.hl7.org/display/FHIR/IG+Publication+Request+Documentation

Reply:
Vu avec @breverseau également, le code "trial-implementation" aligné à IHE ne passe pas ici